profile/profile-gen

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+jq -s -f ./exec.jq --slurpfile config exec-config.json "$1"
```

Comment:
`exec-config.json` doesn't exist, so this command fails

Reply:
leftover from my branch with control of which env goes into the pipeline.